### PR TITLE
Fixes #3277 Forum notification

### LINKED ISSUE #3277: new topic in forum - not sending email
I tested this in NOTIFY area,  email was sent, no problem

but in reality no mail came if new topic was created

**Test email** for new topic:  

file: **mailoutlog.txt**

```
=====13:53:29 18.07.13----------------------------------------------------------------=====
  Mailer opened by Jimako - ID: . Subject: e107.sk: Forum   Log action: 2
-----------------------------------------------------------
13:53:29 18.07.13 - Send to Jimako at janamicek@gmail.com Mail-ID= - Success
```

after it the user created topic
- no change in this file, no email for mainadmin

when I checked this file, it sends email only via Mention plugin from forum.

### PR DESCRIPTION
**Pull request by SimSync**
**Implemented events for:**
- thread created
- thread of new user created
- thread moved
- thread updated
- thread deleted
- post created
- post updated
- post deleted
- post reported

**Deactivated:**
- thread split

### CODE REVIEW

File: e107_plugins/forum/languages/English/English_notify.php [hunk 1]
```diff
@@ -7,11 +7,29 @@
...
+define("LAN_FORUM_NT_TOPIC_DELETED", "Topic deleted");
+define("LAN_FORUM_NT_TOPIC_MOVED", "Topic moved");
+define("LAN_FORUM_NT_POST_CREATED", "Post created");
+define("LAN_FORUM_NT_POST_UPDATED", "Post updated");
 define("LAN_FORUM_NT_POST_DELETED", "Post deleted");
 define("LAN_FORUM_NT_POST_REPORTED", "Post reported");
 
+
+define("LAN_FORUM_NT_NEWTOPIC_MSG", "New thread in forum [f] created by: [u]<br/>Subject: <a href=\"[l]\">[s]</a><br/><br/>Message:<br/>[m]<br/><br/>");
+define("LAN_FORUM_NT_NEWTOPIC_PROB_MSG", "New thread in forum [f] created by new user: [u]<br/>Subject: <a href=\"[l]\">[s]</a><br/><br/>Message:<br/>[m]<br/><br/>");
```
Comment (CaMer0n) [inline e107_plugins/forum/languages/English/English_notify.php]: Great work! Only thing is that LANs should never contain HTML, and should definitely never allow a translator to modify a link URL.  "Subject: [x]" should be enough.

**Comment by SimSync** [comment]
@CaMer0n Just moved the links from the lan file to e_notify.php

**Comment by CaMer0n** [comment]
@SimSync Great! Thank you. We just need to replace all <br /> with [br] OR better yet, split the messages across multiple LANs and put the <br /> in the e_notify itself.

**Comment by SimSync** [comment]
@CaMer0n It was splitted in several lans before, but I changed it to a single lan to hav real sentence. I think it is easier this way also for translating as different languages have different structure in sentences. I'm gonna change the `<br>` to `[br] ` and remove the unused `<br>` at the end.

**Comment by SimSync** [comment]
- Updated lans (removed html)
- Renamed lan vars
- Cleanup code in e_notify.php
- Fixed wrong event name in forum_class.php

**Pull request merged**